packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -109,7 +109,7 @@ export const jiraRemoteProject = (
             sendToSentry(projectRes, {userId, tags: {teamId, projectKey}})
             return null
           }
-          return projectRes
+          return {...projectRes, cloudId, userId, teamId}
```

Comment:
`cloudId`, `userId`, and `teamId` are all required fields on `remoteJiraProject` so I think we should return them from the dataLoader. They're needed in `JiraIssue` [here](https://github.com/ParabolInc/parabol/blob/6f13ec0e76efac0112db7123974c3f370a85593b/packages/server/graphql/types/JiraIssue.ts#L67)

Reply:
I think we shouldn't return those here because it's the input to this loader.
```
export interface JiraRemoteProjectKey {
  userId: string
  teamId: string
  cloudId: string
  projectKey: string
}
```
There shouldn't be a need to return the input unchanged. It's easy enough to add these on the call side.

Reply:
Yeah, it feels strange returning the inputs untouched, but I also don't like calling `jiraRemoteProject` and not getting the non-nullable fields `cloudId`, `teamId`, or `userId`. I don't feel strongly either way so I've removed the inputs from the loader and added them to JiraIssue.project.